roles/openshift_hosted/tasks/registry/secure.yml

```diff
@@ -20,11 +20,11 @@
   failed_when: false
 
 - name: Retrieve registry service IP
-  command: >
-    {{ openshift.common.client_binary }} get service docker-registry
-    -o jsonpath='{.spec.clusterIP}'
-    --config={{ openshift_hosted_kubeconfig }}
-    -n default
+  oc_service:
+    namespace: default
+    name: docker-registry
+    state: list
+    kubeconfig: "{{ openshift_hosted_kubeconfig }}"
```

Comment:
openshift_hosted_kubeconfig is currently a copy of /etc/origin/master/admin.kubeconfig that we make in roles/openshift_hosted/tasks/main.yml. With the module already making a copy of the kubeconfig provided, it doesn't really make sense to make a copy of a copy.